docs/tutorials/01_quadratic_program.ipynb

```diff
@@ -63,7 +63,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You start with an empty model. How to add variables and constraints to a model is explained in the section \"Directly constructing a `QuadraticProgram`\"."
+    "You start with an empty model. How to add variables and constraints to a model is explained in the section [Directly constructing a `QuadraticProgram`](#Directly-constructing-a-QuadraticProgram)."
```

Comment:
I was viewing this file and the link seemed not to work as expected. It certainly fails to come out in the html documentation where I'd argue its more important there than in the source 
![image](https://user-images.githubusercontent.com/40241007/168169278-c1c9887c-9cbd-4103-9238-096e715850d6.png)

Reply:
Thanks. Avoid using the code format in the link (`QuadraticProgram` -> QuadraticProgram) resolves the issue.
![image](https://user-images.githubusercontent.com/31178928/168976745-16c749b3-d6db-44ff-99c6-ceb48a4e0cd7.png)